v2/contracts/linea/LineaSelfVerifier.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {EVMRequest, EVMProver, ProofSequence} from "../EVMProver.sol";
+import {LineaTrieHooks} from "./LineaTrieHooks.sol";
+
+contract LineaSelfVerifier {
+
+	function verify(EVMRequest memory req, bytes32 stateRoot, bytes[] memory proofs, bytes memory order) external view returns (bytes[] memory outputs, uint8 exitCode) {
+		return EVMProver.evalRequest(req, ProofSequence(0, 
+			stateRoot,
+			proofs, order,
+			LineaTrieHooks.proveAccountState,
+			LineaTrieHooks.proveStorageValue
+		));
+	}
+
+	function proveAccountState(bytes32 stateRoot, address target, bytes memory encodedProof) external pure returns (bytes32) {
```

Comment:
What are these here for? They don't seem to be massaging any signatures or anything..

Reply:
they look like leftovers from testing the negative proofs